### ISSUE TITLE
precision issue on android firefox 

### ISSUE THREAD

**Issue by unrealwill** [tensorflow/tfjs#1209]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
tfjs@0.14.2
#### Browser version
firefox 65.0 android on a one+ one hardware.
#### Describe the problem or feature request
I have different result on android than I have on my desktop browser.
The model is a simple keras model specified as tf.float32 input, that contain only a  Conv1d input dim 513, output dim 128, kernel size of 1 
On desktop 
[[[2.2965181, 1.1961352, 2.0413816, ...
On mobile 
[[[2.2964766, 1.196128, 2.0413535, ...

It seems to be consistent with numerical issues linked to doing the computation in half_float (16bits) precision.

In models with more layers, this numerical error adds up, and it becomes total garbage.
How can I force the computation to be done with 32-bits float?
How can I detect the available precision?
Can you point to a tutorial/paper explaining how to modify the architecture/ port the model and retain a semblance of performance?
How can I reduce the precision on the desktop to see what the result would look like on mobile ?

#### Code to reproduce the bug / link to feature request
I called it with a tf.ones as input.

Thanks

**Comment by buaadf** [comment]
seem problem

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
